Configurations/10-main.conf

```diff
@@ -1293,8 +1293,8 @@ my %targets = (
         inherit_from     => [ "VC-common" ],
         template         => 1,
         CFLAGS           => add(picker(debug   => '/Od',
-                                       release => '/O2')),
-        cflags           => add(picker(default => '/Gs0 /GF /Gy',
+                                       release => '/O2 /GF')),
```

Comment:
I think this is unnecessary as /O2 enables /GF anyway, [see here](https://docs.microsoft.com/en-us/cpp/build/reference/gf-eliminate-duplicate-strings?view=vs-2019).

Reply:
I agree so would eliminate /GF altogether, since it is implied by /O2.